### PR TITLE
Release version 0.4.0 of docs

### PR DESCRIPTION
**Pull request by OisinKyne**
Still needs improvement for getting ready for testnets. Started to bring in the new assets gradually.

### CODE REVIEW

File: docs/dv/08_distributed-validator-cluster-manifest.md [hunk 1]
```diff
@@ -2,59 +2,66 @@
...
+The above `cluster_definition.json` is provided as input to the DKG which generates keys and the `cluster_lock.json` file.
+
+The `cluster_lock.json` has the following schema:
+```json
+{
+  "cluster_definition": {...},                              // Cluster definiition json, identical schema to above,
+  "distributed_validators": [                               // Length equaled to num_validators.
+    {
+      "distributed_public_key":  "0x123..abfc",             // DV root pubkey
+      "threshold_verifiers": [ "oA8Z...2XyT", "g1q...icu"], // length of threshold
```
Comment (xenowits) [inline docs/dv/08_distributed-validator-cluster-manifest.md:60]: not sure of this...i think we would have pub_shares here

**Review by xenowits — APPROVED** [review]
Looks good! Couldn't go thoroughly though

File: docs/dv/08_distributed-validator-cluster-manifest.md (hunk 1 above)
Comment (OisinKyne) [reply docs/dv/08_distributed-validator-cluster-manifest.md:60]: Yeah this will be changed in the next version, don't think its in charon yet.

Comment (xenowits) [reply docs/dv/08_distributed-validator-cluster-manifest.md:60]: yeah makes sense

**Pull request merged**